### PR TITLE
PLAT-907-3 Big Renaming and Moving of Core Module Classes

### PR DESCRIPTION
**Pull request by oliver-richers**
Pure renaming, plus adaption of messages.

New permission names:
image

Moved permission classes into new package:
image

This package looks a bit bloated, maybe rename AGModuleInstance to AGModuleInstanceBase and move to subpackage?
image

### CODE REVIEW

File: platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java [hunk 1]
```diff
@@ -473,10 +473,10 @@ public interface CoreI18n extends EmfI18n {
 		.de("Program-UUID fehlt.");
 	I18n0 MODULE = new I18n0("Module")//
 		.de("Modul");
-	I18n1 MODULE_ARG1_IS_NOT_A_STANDARD_MODULE = new I18n1("Module '%s' is not a standard module.")//
-		.de("Modul '%s' ist kein Standard-Modul.");
 	I18n0 MODULE_CLASS = new I18n0("Module Class")//
 		.de("Modulklasse");
+	I18n2 MODULE_CLASS_ARG1_MUST_IMPLEMENT_INTERFACE_ARG2 = new I18n2("Module class '%s' must implement interface '%s'.")//
+		.de("Modulklasse '%s' muss das Inerface '%s' implementieren.");
```
[the remark this answers is not in the document]
Comment (oliver-richers) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java:479]: changed

File: platform-core-module/src/main/java/com/softicar/platform/core/module/module/instance/ModuleClassNotImplementingModuleInterfaceException.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.softicar.platform.core.module.module.instance;
+
+import com.softicar.platform.common.core.exceptions.SofticarDeveloperException;
+import com.softicar.platform.core.module.module.IModule;
+import com.softicar.platform.emf.module.IEmfModule;
+
+public class ModuleClassNotImplementingModuleInterfaceException extends SofticarDeveloperException {
```
[the remark this answers is not in the document]
Comment (oliver-richers) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/module/instance/ModuleClassNotImplementingModuleInterfaceException.java:7]: renamed

[the remark this answers is not in the document]
Comment (oliver-richers) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/module/instance/ModuleClassNotImplementingModuleInterfaceException.java:7]: did so

File: platform-core-module/src/test/java/com/softicar/platform/core/module/test/module/standard/alpha/TestModuleAlpha.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package com.softicar.platform.core.module.test.module.standard.alpha;
```
[the remark this answers is not in the document]
Comment (oliver-richers) [reply platform-core-module/src/test/java/com/softicar/platform/core/module/test/module/standard/alpha/TestModuleAlpha.java:1]: fixed

**Review by alexander-schmidt — APPROVED** [review]

**Pull request merged**